### PR TITLE
Update wording in the tutorials

### PR DESCRIPTION
**Pull request by psteinb**
While I was working through the tutorial (which are great!), I took the liberty of refining some wording. I very few mistakes of English language. I also added some bits here and there to make the tutorials more readable (for someone like me). I hope you find this useful.

### CODE REVIEW

**Comment by psteinb** [comment]
Note, I used `nbterm` to edit the notebooks. Notebooks are not great for diffing and creating PRs in general when opened with jupyter. In any case, I observed that `nbterm` replaced some characters with unicode equivalents. Whoever reviews this PR, please have a look at the notebooks locally first. They view fine on my box, but you never know.

**Review by francois-rozet — COMMENTED** [review]
Hello @psteinb, thanks for the feedback and fixes! I agree with most of your additions but made a few wording suggestions.

I still have to check that the UTF-8 edits display correctly in the docs and locally.

File: docs/tutorials/basics.ipynb [hunk 1]
```diff
@@ -94,7 +101,7 @@
     "\n",
     "$$ p(X = x) = p(Z = f(x)) \\left| \\det \\frac{\\partial f(x)}{\\partial x} \\right| $$\n",
     "\n",
-    "and sampling from $p(X)$ can be performed by first drawing realizations $z \\sim p(Z)$ and then applying the inverse transformation $x = f^{-1}(z)$. Such combination of a base distribution and a bijective transformation is sometimes called a *normalizing flow* as the base distribution is often standard normal."
+    "and sampling from $p(X)$ can be performed by first drawing realizations $z \\sim p(Z)$ and then applying the inverse transformation $x = f^{-1}(z)$. Such combination of a base distribution and a bijective transformation is sometimes called a *normalizing flow*. The name indicates that the base distribution is a standard *normal* distribution."
```
[the remark this answers is not in the document]
Comment (psteinb) [reply docs/tutorials/basics.ipynb:104]: fair enough. I convoluted simpler language with a too strong statement. How about this:

```suggestion
    "and sampling from $p(X)$ can be performed by first drawing realizations $z \\sim p(Z)$ and then applying the inverse transformation $x = f^{-1}(z)$. Such combination of a base distribution and a bijective transformation is sometimes called a *normalizing flow*. The term 'normalizing' indicates that the base distribution is often a standard *normal* distribution."
```

File: docs/tutorials/basics.ipynb [hunk 2]
```diff
@@ -139,7 +146,7 @@
    "source": [
     "### Variational inference\n",
     "\n",
-    "Let's say we have a dataset of pairs $(x, c) \\sim p(X, C)$ and want to model the distribution of $X$ given $c$, that is $p(X | c)$. The goal of variational inference is to find the model $q_{\\phi^\\star}(X | c)$ that is most similar to $p(X | c)$ among a family of (conditional) distributions $q_\\phi(X | c)$ distinguished by their parameters $\\phi$. Expressing the dissimilarity between two distributions as their [Kullback-Leibler](https://wikipedia.org/wiki/Kullback–Leibler_divergence) (KL) divergence, the variational inference objective becomes\n",
+    "Let's say we have a dataset of pairs $(x, c) \\sim p(X, C)$ and want to model the distribution of $X$ given $c$, that is $p(X | c)$. The goal of variational inference is to find the model $q_{\\phi^\\star}(X | c)$ that is most similar to $p(X | c)$ among a family of (conditional) distributions $q_\\phi(X | c)$ distinguished by their parameters $\\phi$. Expressing the dissimilarity between two distributions as their [Kullback-Leibler](https://wikipedia.org/wiki/Kullback\u2013Leibler_divergence) (KL) divergence, the variational inference objective becomes\n",
```
[the remark this answers is not in the document]
Comment (psteinb) [reply docs/tutorials/basics.ipynb:149]: Thanks for checking.

File: docs/tutorials/forward_kl.ipynb [hunk 3]
```diff
@@ -305,6 +305,7 @@
     }
    ],
    "source": [
+    "# sample the flow while conditioning on the 'top' part of two moons\n",
```
[the remark this answers is not in the document]
Comment (psteinb) [reply docs/tutorials/forward_kl.ipynb:308]: This is too brief for me and can be misunderstood easily.

File: docs/tutorials/forward_kl.ipynb [hunk 4]
```diff
@@ -329,6 +330,7 @@
     }
    ],
    "source": [
+    "# sample the flow while conditioning on the 'bottom' part of two moons\n",
```
[the remark this answers is not in the document]
Comment (psteinb) [reply docs/tutorials/forward_kl.ipynb:333]: see above with the top part

File: docs/tutorials/basics.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (francois-rozet) [reply docs/tutorials/basics.ipynb:104]: How about

> The term *normalizing* refers to the fact that the base distribution is often chosen as a (standard) normal distribution.

File: docs/tutorials/forward_kl.ipynb (hunk 3 above)
[the remark this answers is not in the document]
Comment (francois-rozet) [reply docs/tutorials/forward_kl.ipynb:308]: Would this be better?

> sample from the flow conditioned on the top moon label

I don't like the term "part" as it does not convey that modes are moon-shaped.

File: docs/tutorials/basics.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (psteinb) [reply docs/tutorials/basics.ipynb:104]: sounds awesome! Just click commit below ...

```suggestion
    "and sampling from $p(X)$ can be performed by first drawing realizations $z \\sim p(Z)$ and then applying the inverse transformation $x = f^{-1}(z)$. Such combination of a base distribution and a bijective transformation is sometimes called a *normalizing flow*. The term *normalizing* refers to the fact that the base distribution is often a (standard) *normal* distribution."
```

File: docs/tutorials/forward_kl.ipynb (hunk 3 above)
[the remark this answers is not in the document]
Comment (psteinb) [reply docs/tutorials/forward_kl.ipynb:308]: Ok, this makes sense then. Yeah, let's go wit this then:
```suggestion
    "# sample from the flow conditioned on the top moon label\n",
```

File: docs/tutorials/forward_kl.ipynb (hunk 4 above)
[the remark this answers is not in the document]
Comment (psteinb) [reply docs/tutorials/forward_kl.ipynb:333]:
```suggestion
    "# sample from the flow conditioned on the bottom moon label\n",
```

File: docs/tutorials/basics.ipynb [hunk 5]
```diff
@@ -324,6 +331,13 @@
     "    optimizer.zero_grad()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note, `model(c)` calls the `forward` method as described above."
+   ]
+  },
```
[the remark this answers is not in the document]
Comment (francois-rozet) [reply docs/tutorials/basics.ipynb:340]: @psteinb I think I misunderstood your comment here. Indeed, it is not clear that `model(c)` calls the forward method. However, I think this note should come earlier. Just before block [7] is probably best.

[the remark this answers is not in the document]
Comment (psteinb) [reply docs/tutorials/basics.ipynb]: I moved this comment in 20e3215

[the remark this answers is not in the document]
Comment (francois-rozet) [reply docs/tutorials/basics.ipynb]: I agree, where would you put this?

**Comment by francois-rozet** [comment]
No problem, thank you for the edits, I think the tutorial is much clearer now!

> Is this still an open issue?

I think its fine like it is now. Feel free to open another PR if you want to improve the tutorials (or add new ones), though!
